tests/test_basic.py

```diff
@@ -5,11 +5,67 @@
 import unittest
```

Comment:
Create a new file with these changes, e.g. test_crash_similarity.py, and remove the changes from this file.